ClinGen/clincoded
# Connects to #732. Population tab

Test instance: http://732-mc-population-tab-try2.instance.clinicalgenome.org/

* Re-structured `populationObj`
* Updated the parsers to support new model
* Revised rendering of tables to use loops
* Re-implemented 1000G parser
* Connected `populationObj` to evaluation/assessment forms

![image](https://cloud.githubusercontent.com/assets/4326866/16395971/6a09b23a-3c71-11e6-9f37-eb0430386eb7.png)

Testing:

1. Load variant (tested with 139214, 139215)
2. Check population tab for proper rendering and linkouts of data
3. Create interpretation and save it
4. Confirm that the interpretation has saved properly, and the interpretation evaluation evidence is also saved properly

Caveats/To-Do's:

* ~~When navigating away from the VCI page, a bunch of `setState` errors pop up. Maybe a separate ticket:~~
![image](https://cloud.githubusercontent.com/assets/4326866/16397008/afb9c856-3c76-11e6-8006-d90bf26c1c6a.png)
  * Fixed as of commit 57b85e1 - test by loading VCI, then pressing the Header image to go to dashboard. Console should not have above errors.
* #730 and #731 not addressed here
* Top section of Populations Tab + CI stuff not implemented yet
